scripts/integration-tests/docker-integration-test.sh

```diff
@@ -44,7 +44,7 @@ curl -vvvsSf -X POST localhost:7201/api/v1/namespace -d '{
 
 echo "Sleep while namespace is init'd"
 
-sleep 60 # TODO Replace sleeps with logic to determine when to proceed
+sleep 10 # TODO Replace sleeps with logic to determine when to proceed
```

Comment:
Why'd you change this?

Reply:
I'd changed it to 60 previously, reverting the change here